test/vault_cluster_enterprise_test.go

```diff
@@ -118,8 +116,9 @@ func testAutoUnseal(t *testing.T, asgNameOutputVar string, sshUserName string, t
 	establishConnectionToCluster(t, initialCluster)
 	waitForVaultToBoot(t, initialCluster)
 
-	logger.Logf(t, "Initializing the cluster")
-	ssh.CheckSshCommand(t, initialCluster.Leader, "vault operator init")
+	retry.DoWithRetry(t, "Initializing the cluster", 10, 10*time.Second, func() (string, error) {
+		return ssh.CheckSshCommandE(t, initialCluster.Leader, "vault operator init")
```

Comment:
`terraform init` was sometimes failing to install aws provider because it just fails to make the http request. I discussed with Josh and he suggested this type of change becomes built-in from terratest, but we should see how it behaves here first.